nixos/modules/services/misc/nscd-sssd.conf

```diff
@@ -0,0 +1,15 @@
+server-user             nscd
+threads                 1
+paranoia                no
+debug-level             0
+
+enable-cache            passwd          no
+enable-cache            group           no
```

Comment:
What happened to me was after PAM login e.g. bash would set my prompt to `I have no name!@nix-vm $` and `whoami` wouldn't work. Also according to glibc sources that seems the intended behaviour:

````
1155       /* Is this service enabled?  */
1156       if (__glibc_unlikely (!db->enabled))
1157         {
1158           /* No, sent the prepared record.  */
1159           if (TEMP_FAILURE_RETRY (send (fd, db->disabled_iov->iov_base,
1160                                         db->disabled_iov->iov_len,
1161                                         MSG_NOSIGNAL))
1162               != (ssize_t) db->disabled_iov->iov_len
1163               && __builtin_expect (debug_level, 0) > 0)
1164             {
1165               /* We have problems sending the result.  */
1166               char buf[256];
1167               dbg_log (_("cannot write result: %s"),
1168                        strerror_r (errno, buf, sizeof (buf)));
1169             }
1170 
1171           return;
1172         }
````
... with `db->enabled` set as:
````
143       else if (strcmp (entry, "enable-cache") == 0)
144         {
145           int idx = find_db (arg1);
146           if (idx >= 0)
147             {
148               if (strcmp (arg2, "no") == 0)
149                 dbs[idx].enabled = 0;
150               else if (strcmp (arg2, "yes") == 0)
151                 dbs[idx].enabled = 1;
152             }
153         }
````